src/containers/WorkspaceRecordingView.tsx

```diff
@@ -0,0 +1,67 @@
+import { Grid } from '@material-ui/core';
```

Comment:
This WorkspaceRecordingView replaces the old RecordingView. It is pretty much the same, but simplified. And it uses `workspaceRouteDispatch` which will be described.